vassal-app/src/main/java/VASSAL/build/module/BasicCommandEncoder.java

```diff
@@ -78,26 +81,62 @@
 import VASSAL.tools.SequenceEncoder;
 
 /**
- * A {@link CommandEncoder} that handles the basic commands: {@link AddPiece},
- * {@link RemovePiece}, {@link ChangePiece}, {@link MovePiece}. If a module
- * defines custom {@link GamePiece} classes, then this class may be overriden
- * and imported into the module. Subclasses should override the
- * {@link #createDecorator} method or, less often, the {@link #createBasic} or
- * {@link #createPiece} methods to allow instantiation of the custom
- * {@link GamePiece} classes.
+ * {@see <a href="http://google.com">http://google.com</a>}
```

Comment:
Er, what?

Reply:
Derp. I was teaching myself how to format hyperlinks :D